marge/job.py

```diff
@@ -56,7 +56,9 @@ def ensure_mergeable_mr(self, merge_request):
         if not approvals.sufficient:
             raise CannotMerge(
                 'Insufficient approvals '
-                '(have: {0.approver_usernames} missing: {0.approvals_left})'.format(approvals)
+                '(have {0} but needs {1})'.format(
```

Comment:
Grammar nitpicking, but shouldn't it be either `have + need` or `has + needs`?

Reply:
Yep, fixed.